public/images/calculate.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Seems to be something to add in the uikit no ?

Reply:
This file is removed